### ISSUE TITLE
Graphs are not available on windows zip build

### ISSUE THREAD

**Issue by DarkFenX** [pyfa-org/Pyfa#2141]
```
[2020-02-25 13:34:16.584954] CRITICAL: graphs.gui.canvasPanel: Exception when importing Matplotlib. Continuing without importing.
[2020-02-25 13:34:16.584954] CRITICAL: graphs.gui.canvasPanel: Traceback (most recent call last):
  File "graphs\gui\canvasPanel.py", line 41, in <module>
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "C:\Python37-x64\lib\site-packages\PyInstaller\loader\pyimod03_importers.py", line 623, in exec_module
  File "site-packages\matplotlib\__init__.py", line 1013, in <module>
  File "site-packages\matplotlib\__init__.py", line 880, in rc_params
  File "site-packages\matplotlib\__init__.py", line 701, in matplotlib_fname
RuntimeError: Could not find matplotlibrc file; your Matplotlib install is broken
```

**Comment by rdcalvo** [comment]
+1

**Comment by rdcalvo** [comment]
Duplicated issue:
Graphs Not Working. #2151

Issue #2151 expands the scope of the bug reporting to other operational systems.

**Comment by DarkFenX** [comment]
Yeah this is an issue which I didn't have time to fix before my vacation. Gonna take a look at it this week

**Comment by DarkFenX** [comment]
I tried rebuilding pyfa locally. Interestingly enough, if I launch distribution right after building it, graphs work. After I compress it by executing `7z a "pyfa-$env:PYFA_VERSION-win.zip" -r "$env:PYFA_DIST_DIR\pyfa\*.*"` and decompress, graphs cannot be launched anymore. So, something is lost during compression-decompression.

**Comment by DarkFenX** [comment]
Some files were not included in zip build. Fixed in 8dc1457ebb599c52ba7d29c0bdf4a267d01cbbe2
